### PR TITLE
[5.5] Ensure return null are consistent.

### PR DESCRIPTION
**Pull request by lucasmichot**
Ensure return null are consistent.

### CODE REVIEW

File: src/Illuminate/Auth/Access/Gate.php [hunk 1]
```diff
@@ -398,7 +398,7 @@ public function getPolicyFor($class)
         }
 
         if (! is_string($class)) {
-            return null;
+            return;
```
Comment (a-komarev) [inline src/Illuminate/Auth/Access/Gate.php]: But `return;` !== null its void, isn't it?

Comment (lucasmichot) [reply src/Illuminate/Auth/Access/Gate.php]: Which is interpreted as null, thus many different return type - as return docblock suggests

Comment (a-komarev) [reply src/Illuminate/Auth/Access/Gate.php]: Thats all about PHP weak types :(

Comment (lucasmichot) [reply src/Illuminate/Auth/Access/Gate.php]: You'll notice that this is the case with all other `return null;` / `(void)`
This PR just enforces consistency.

**Pull request merged**